src/pylorax/imgutils.py

```diff
@@ -397,8 +408,12 @@ def __exit__(self, exc_type, exc_value, traceback):
 
 ######## Functions for making filesystem images ##########################
 
-def mkfsimage(fstype, rootdir, outfile, size=None, mkfsargs=[], mountargs="", graft={}):
+def mkfsimage(fstype, rootdir, outfile, size=None, mkfsargs=None, mountargs="", graft=None):
     '''Generic filesystem image creation function.
+    if mkfsargs is None:
```

Comment:
Oops, this ended up in the middle of a comment block. Move it past the ''' on line 420 :)